scripts/deployment/DeploySideChainGovernance.s.sol

```diff
@@ -23,27 +23,29 @@ contract DeploySideChainGovernance is Utils {
     TimelockControllerWithCounter public timelock;
 
     function run() external {
-        uint256 deployerPrivateKey = vm.deriveKey(vm.envString("MNEMONIC_POLYGON"), "m/44'/60'/0'/0/", 0);
+        uint256 deployerPrivateKey = vm.deriveKey(vm.envString("MNEMONIC_MAINNET"), "m/44'/60'/0'/0/", 0);
         vm.startBroadcast(deployerPrivateKey);
         address deployer = vm.addr(deployerPrivateKey);
         vm.label(deployer, "Deployer");
 
         // TODO can be modified to deploy on any chain
         uint256 srcChainId = CHAIN_ETHEREUM;
-        uint256 destChainId = CHAIN_LINEA;
+        uint256 destChainId = CHAIN_MODE;
+        address destSafeMultiSig = 0x7DE8289038DF0b89FFEC71Ee48a2BaD572549027; // guardian
```

Comment:
We don't have it in the sdk?

Reply:
so far not as I will update SDK once everything is deployed